server/form-pages/utils/questions.ts

```diff
@@ -411,10 +411,10 @@ export const getQuestions = (name: string) => {
       'funding-source': {
         fundingSource: {
           question: `How will ${name} pay for their accommodation and service charge?`,
+          hint: 'Applicants must pay for a weekly service charge using their personal money or wages. This service charge is not eligible to be covered by Housing Benefit.',
           answers: {
-            personalSavings: 'Personal money or savings',
-            benefits: 'Benefits',
-            both: 'Both',
+            personalSavings: 'Personal money or wages',
```

Comment:
hmm not sure why its no longer matching on 'Personal money' but looks like the e2e tests are failing?

Reply:
it was matching on both radios 🫠 thanks though, should be fixed now